lib/ansible/config/routing.yml

```
@@ -1721,869 +1721,869 @@ plugin_routing:
     edgeos_config:
-      redirect: community.general.edgeos_config
+      redirect: community.network.edgeos_config
     edgeos_facts:
-      redirect: community.general.edgeos_facts
+      redirect: community.network.edgeos_facts
     edgeswitch_facts:
-      redirect: community.general.edgeswitch_facts
+      redirect: community.network.edgeswitch_facts
     edgeswitch_vlan:
-      redirect: community.general.edgeswitch_vlan
+      redirect: community.network.edgeswitch_vlan
     enos_command:
-      redirect: community.general.enos_command
+      redirect: community.network.enos_command
     enos_config:
-      redirect: community.general.enos_config
+      redirect: community.network.enos_config
     enos_facts:
-      redirect: community.general.enos_facts
+      redirect: community.network.enos_facts
     eric_eccli_command:
-      redirect: community.general.eric_eccli_command
+      redirect: community.network.eric_eccli_command
     exos_command:
-      redirect: community.general.exos_command
+      redirect: community.network.exos_command
     exos_config:
-      redirect: community.general.exos_config
+      redirect: community.network.exos_config
     exos_facts:
-      redirect: community.general.exos_facts
+      redirect: community.network.exos_facts
     exos_l2_interfaces:
-      redirect: community.general.exos_l2_interfaces
+      redirect: community.network.exos_l2_interfaces
     exos_lldp_global:
-      redirect: community.general.exos_lldp_global
+      redirect: community.network.exos_lldp_global
     exos_lldp_interfaces:
-      redirect: community.general.exos_lldp_interfaces
+      redirect: community.network.exos_lldp_interfaces
     exos_vlans:
-      redirect: community.general.exos_vlans
+      redirect: community.network.exos_vlans
     bigip_asm_policy:
-      redirect: community.general.bigip_asm_policy
+      redirect: f5networks.f5_nodules.bigip_asm_policy
     bigip_device_facts:
-      redirect: community.general.bigip_device_facts
+      redirect: f5networks.f5_nodules.bigip_device_facts
     bigip_facts:
-      redirect: community.general.bigip_facts
+      redirect: f5networks.f5_nodules.bigip_facts
     bigip_gtm_facts:
-      redirect: community.general.bigip_gtm_facts
+      redirect: f5networks.f5_nodules.bigip_gtm_facts
     bigip_iapplx_package:
-      redirect: community.general.bigip_iapplx_package
+      redirect: f5networks.f5_nodules.bigip_iapplx_package
     bigip_security_address_list:
-      redirect: community.general.bigip_security_address_list
+      redirect: f5networks.f5_nodules.bigip_security_address_list
     bigip_security_port_list:
-      redirect: community.general.bigip_security_port_list
+      redirect: f5networks.f5_nodules.bigip_security_port_list
     bigip_traffic_group:
-      redirect: community.general.bigip_traffic_group
+      redirect: f5networks.f5_nodules.bigip_traffic_group
     bigiq_device_facts:
       redirect: community.general.bigiq_device_facts
```

```suggestion
    bigiq_device_facts:
      redirect: community.general.bigiq_device_facts
```
i don't see this module in community.general in master.
I found bigiq_device_info.py there https://github.com/F5Networks/f5-ansible/tree/devel/library/modules

---

Both _facts and _info are in community.network as well (_facts is a symlink to _info). I guess we can remove both, and instead redirect this to the _info module in the F5 collection.

---

Good spot, removed from c.network in https://github.com/ansible-collections/community.network/pull/46